src/components/CookieNotice/CookieNotice.tsx

```diff
@@ -0,0 +1,97 @@
+import "./CookieNotice.scss";
+import React from "react";
+import Backdrop from "@material-ui/core/Backdrop";
+import CookiePolicy from "./CookiePolicy";
+
+const COOKIE_CONSENT_NAME = "scrumlr_cookieConsent";
+
+const CookieNotice = () => {
+  // states whether cookie notice is shown.
+  const [showCookieNotice, setShowCookieNotice] = React.useState<boolean>(true);
```

Comment:
isn't <boolean>(true) cleaner, i.e. more explicit?

Reply:
1. You could import useState so you don't have to write `React.useState(...)`, but that's up to you
2. You don't need to write `useState<boolean>(true)` because the initialValue already applies the boolean type to the function